templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -97,12 +97,12 @@
           </tr>
           <tr class="crm-payment-form-block-fee_amount">
             <td class="label">{$form.fee_amount.label}</td>
-            <td{$valueStyle}>{$form.fee_amount.html}</td>
+            <td>{$form.fee_amount.html}</td>
           </tr>
         </table>
       </div>
       {/if}
-      {include file='CRM/Core/BillingBlockWrapper.tpl'}
+      {include file='CRM/Core/BillingBlockWrapper.tpl' currency=false}
```

Comment:
we pass currency as '' for the template variable as it  retrieves it from the form field on this form - so we just need to avoid the notice 
![image](https://github.com/civicrm/civicrm-core/assets/336308/f3d8037a-063a-4933-84aa-cd22e5aa5243)

![image](https://github.com/civicrm/civicrm-core/assets/336308/e7839d23-b6f7-414f-a2fa-1854e977811a)